manager/recurringjob.go

```diff
@@ -69,14 +69,16 @@ func (m *VolumeManager) UpdateRecurringJob(spec longhorn.RecurringJobSpec) (*lon
 		reflect.DeepEqual(recurringJob.Spec.Groups, spec.Groups) &&
 		recurringJob.Spec.Retain == spec.Retain &&
 		recurringJob.Spec.Concurrency == spec.Concurrency &&
-		reflect.DeepEqual(recurringJob.Spec.Labels, spec.Labels) {
+		reflect.DeepEqual(recurringJob.Spec.Labels, spec.Labels) &&
+		reflect.DeepEqual(recurringJob.Spec.Parameters, spec.Parameters) {
```

Comment:
nit: why not directly compare the entire spec?

Reply:
fixed.